examples/feature_demo/transparency1.py

```diff
@@ -34,43 +43,40 @@
 camera = gfx.OrthographicCamera(100, 100)
 
 scene_overlay = gfx.Scene()
-blend_mode_text = gfx.Text(
-    text=f"Blend mode: {renderer.blend_mode}",
+blend_text = gfx.Text(
+    text=f"Blending: {plane1.material.blending['name']}",
     anchor="bottom-left",
     material=gfx.TextMaterial(outline_thickness=0.3),
 )
-scene_overlay.add(blend_mode_text)
+scene_overlay.add(blend_text)
 
 screen_camera = gfx.ScreenCoordsCamera()
 
 
 @renderer.add_event_handler("key_down")
 def handle_event(event):
+    canvas.request_draw()
     if event.key == " ":
         print("Rotating scene element order")
         scene.add(scene.children[1])  # skip bg
-        canvas.request_draw()
     elif event.key == ".":
         clr = "#fff" if background.material.color_bottom_left == "#000" else "#000"
         print(f"Changing background color to {clr}")
         background.material.set_colors(clr)
-        canvas.request_draw()
-    elif event.key in "012345678":
+    elif event.key in "12345":
         m = [
-            None,  # 0
-            "opaque",  # 1
-            "dither",  # 2
-            "ordered1",  # 3
-            "ordered2",  # 4
+            None,
+            "no",  # 1
+            "normal",  # 2
+            "additive",  # 3
+            "dither",  # 4
             "weighted",  # 5
-            "weighted_depth",  # 6
-            "weighted_plus",  # 7
```

Comment:
I think it would be interesting to "rebuild" this with the `image_stitching` ideas.

On clear win already from this new architecture is that the text is always beautiful.